packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js

```diff
@@ -1,70 +1,43 @@
 import filter from 'lodash/filter';
-import map from 'lodash/map';
+import get from 'lodash/get';
+
+const REGIONS = {
+  EU: 'EUROPE',
+  CA: 'CANADA',
+  US: 'USA',
+};
 
 export default /* @ngInject */ ($stateProvider, coreConfigProvider) => {
-  if (!coreConfigProvider.isRegion('US')) {
-    $stateProvider.state('pci.projects.project.failover-ips.onboarding.order', {
-      url: '/order',
-      layout: 'modal',
-      views: {
-        modal: {
-          component: 'pciProjectFailoverIpsLegacyOrder',
-        },
+  $stateProvider.state('pci.projects.project.failover-ips.onboarding.order', {
+    url: '/order',
+    layout: 'modal',
+    views: {
+      modal: {
+        component: 'pciProjectFailoverIpsOrderOrder',
       },
-      resolve: {
-        breadcrumb: /* @ngInject */ () => null,
-        regions: /* @ngInject */ (OvhApiCloudProjectRegion, projectId) =>
-          OvhApiCloudProjectRegion.v6()
-            .query({
-              serviceName: projectId,
-            })
-            .$promise.then((regions) =>
-              Promise.all(
-                map(
-                  regions,
-                  (id) =>
-                    OvhApiCloudProjectRegion.v6().get({
-                      serviceName: projectId,
-                      id,
-                    }).$promise,
+    },
+    resolve: {
+      breadcrumb: /* @ngInject */ () => null,
+      /* @ngInject */
+      products: (OvhApiOrderCatalogFormatted) =>
```

Comment:
```suggestion
      products: /* @ngInject */ (OvhApiOrderCatalogFormatted) =>
```
shouldn't be this way ?